<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1354</width>
    <height>724</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Krishna Kids' Educational Point</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1341</width>
      <height>671</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tabWelcome">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Welcome</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>931</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lOrgName">
         <property name="font">
          <font>
           <family>Economica</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;KRISHNA KIDS' EDUCATIONAL POINT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCalendarWidget" name="calendarWidget"/>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lSelection">
             <property name="font">
              <font>
               <family>Economica</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Select Task :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxStudents">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QTreeWidget" name="twStudents">
                <property name="font">
                 <font>
                  <family>Ubuntu</family>
                  <pointsize>11</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="midLineWidth">
                 <number>0</number>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="tabKeyNavigation">
                 <bool>true</bool>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="textElideMode">
                 <enum>Qt::ElideRight</enum>
                </property>
                <property name="autoExpandDelay">
                 <number>-1</number>
                </property>
                <property name="uniformRowHeights">
                 <bool>false</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <property name="animated">
                 <bool>true</bool>
                </property>
                <property name="allColumnsShowFocus">
                 <bool>false</bool>
                </property>
                <property name="expandsOnDoubleClick">
                 <bool>false</bool>
                </property>
                <property name="columnCount">
                 <number>1</number>
                </property>
                <column>
                 <property name="text">
                  <string>Students</string>
                 </property>
                 <property name="font">
                  <font>
                   <family>Economica</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="background">
                  <color>
                   <red>31</red>
                   <green>126</green>
                   <blue>166</blue>
                  </color>
                 </property>
                </column>
                <item>
                 <property name="text">
                  <string>Admission</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fee Deposite</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Grade Submission</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Show Profile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Update Profile</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbStudents">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxFaculty">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QTreeWidget" name="twFaculty">
                <property name="font">
                 <font>
                  <family>Ubuntu</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="midLineWidth">
                 <number>0</number>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="tabKeyNavigation">
                 <bool>true</bool>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="textElideMode">
                 <enum>Qt::ElideRight</enum>
                </property>
                <property name="autoExpandDelay">
                 <number>-1</number>
                </property>
                <property name="uniformRowHeights">
                 <bool>false</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <property name="animated">
                 <bool>true</bool>
                </property>
                <property name="allColumnsShowFocus">
                 <bool>false</bool>
                </property>
                <property name="expandsOnDoubleClick">
                 <bool>false</bool>
                </property>
                <property name="columnCount">
                 <number>1</number>
                </property>
                <column>
                 <property name="text">
                  <string>Faculty</string>
                 </property>
                 <property name="font">
                  <font>
                   <family>Economica</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="background">
                  <color>
                   <red>31</red>
                   <green>126</green>
                   <blue>166</blue>
                  </color>
                 </property>
                </column>
                <item>
                 <property name="text">
                  <string>Add Profile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Remove Profile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Renumeration</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Show Profile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Update Profile</string>
                 </property>
                </item>
               </widget>
              </item>
              <item alignment="Qt::AlignBottom">
               <widget class="QPushButton" name="pbFaculty">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxAccounts">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QTreeWidget" name="twAccounts">
                <property name="font">
                 <font>
                  <family>Ubuntu</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="midLineWidth">
                 <number>0</number>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="tabKeyNavigation">
                 <bool>true</bool>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="textElideMode">
                 <enum>Qt::ElideRight</enum>
                </property>
                <property name="autoExpandDelay">
                 <number>-1</number>
                </property>
                <property name="uniformRowHeights">
                 <bool>false</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <property name="animated">
                 <bool>true</bool>
                </property>
                <property name="allColumnsShowFocus">
                 <bool>false</bool>
                </property>
                <property name="expandsOnDoubleClick">
                 <bool>false</bool>
                </property>
                <property name="columnCount">
                 <number>1</number>
                </property>
                <column>
                 <property name="text">
                  <string>Accounts</string>
                 </property>
                 <property name="font">
                  <font>
                   <family>Economica</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="background">
                  <color>
                   <red>31</red>
                   <green>126</green>
                   <blue>166</blue>
                  </color>
                 </property>
                </column>
                <item>
                 <property name="text">
                  <string>Monthly View</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Yearly View</string>
                 </property>
                </item>
               </widget>
              </item>
              <item alignment="Qt::AlignBottom">
               <widget class="QPushButton" name="pbAccounts">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1354</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>twStudents</tabstop>
  <tabstop>pbStudents</tabstop>
  <tabstop>twFaculty</tabstop>
  <tabstop>pbFaculty</tabstop>
  <tabstop>twAccounts</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>twStudents</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pbStudents</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1081</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>1167</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twFaculty</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pbFaculty</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1081</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>1167</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twAccounts</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pbAccounts</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1081</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>1162</x>
     <y>651</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
